aldryn_newsblog/menu.py

```diff
@@ -22,7 +22,8 @@ class NewsBlogMenu(CMSAttachMenu):
     def get_queryset(self, request):
         """Returns base queryset with support for preview-mode."""
         queryset = Article.objects
-        if not (request.toolbar and request.toolbar.edit_mode):
+        if not (hasattr(request, "toolbar") and
+                getattr(request.toolbar, "edit_mode", False)):
```

Comment:
In a new version of Django cms we changed this and newsblog has this code to check edit_mode for old and new django-cms
https://github.com/divio/aldryn-newsblog/blob/master/aldryn_newsblog/compat.py
also:
https://github.com/divio/django-cms/blob/develop/cms/toolbar/toolbar.py#L81
https://github.com/divio/django-cms/blob/develop/cms/toolbar/toolbar.py#L493

but checking hasattr toolbar in a request is good in my opinion